src/main/proto/org/dependencytrack/notification/v1/notification.proto

```diff
@@ -80,6 +81,13 @@ message BomProcessingFailedSubject {
   string token = 4;
 }
 
+message BomValidationFailedSubject {
+  Project project = 1;
+  Bom bom = 2;
+  repeated string errors = 3;
+  string format = 4;
```

Comment:
Do we need this? The `Bom` message already has a `format` field.

Reply:
yes mom already has format info, it can be removed 👍🏼